### DISCUSSION TITLE
Release v1.5.3 / 5.60.3

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2440] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build fixes various issues and minor improvements.

**Note:** if you encounter the new message **SBIE2227**, the right course of action is to enable 8.3 name support for all volume on your system. Use the following command in an elevated command prompt:

```
fsutil behavior set disable8dot3 0
```

This is not a new issue, but an old forgotten one. 
The message has been added to bring attention to potential problems and offer instructions how to properly resolve the issue.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

# Changelog

### Added
- boxes set to auto-delete are now marked with a small red recycle symbol
- added Vietnamese language on Plus UI [#2438](https://github.com/sandboxie-plus/Sandboxie/pull/2438)

### Fixed
- fixed issue with box options [#2400](https://github.com/sandboxie-plus/Sandboxie/issues/2400)
- fixed issue with Smart App Control [#2341](https://github.com/sandboxie-plus/Sandboxie/issues/2341)
- fixed issue with snapshots when using privacy boxes [#2427](https://github.com/sandboxie-plus/Sandboxie/issues/2427)
- fixed issue with m_pColorslider change not applied [#2433](https://github.com/sandboxie-plus/Sandboxie/pull/2433) (thanks okrc)
- fixed issue with switching snapshots when the file panel is open
- fixed issue with file panel when an empty box is selected [#2419](https://github.com/sandboxie-plus/Sandboxie/issues/2419)
- fixed issue with menu icon not being disabled [#2406](https://github.com/sandboxie-plus/Sandboxie/issues/2406)
- fixed compatibility issue with Microsoft Edge [#2312](https://github.com/sandboxie-plus/Sandboxie/issues/2312)
- fixed UsePrivacyMode=y compatibility with Windows 7 [#2423](https://github.com/sandboxie-plus/Sandboxie/issues/2423)
- fixed minor issue during Sandboxie Plus uninstall [#2421](https://github.com/sandboxie-plus/Sandboxie/issues/2421)
- fixed BSOD issue when driver initialization fails (introduced in 1.5.1) [#2431](https://github.com/sandboxie-plus/Sandboxie/issues/2431)
- fixed fake paths being listed even if true paths do not exist [#2403](https://github.com/sandboxie-plus/Sandboxie/issues/2403)
- fixed issue with Firefox 106.x requesting write access to plugin executables [#2391](https://github.com/sandboxie-plus/Sandboxie/issues/2391) [#2411](https://github.com/sandboxie-plus/Sandboxie/issues/2411)

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.5.3'>Release v1.5.3 / 5.60.3</a>.</em>

**Comment by BilleBarrett** [comment]
Thank you to all of you that contributed to this great product
Since I have installed this on almost all of my client's computers, malware has not been a big problem for me
Bill e. Barrett
Remote Desktop Services

**Reply by DavidXanatos** [reply]
How about getting each of your clients to buy a business certificate? That would be very much appreciated and for commercial users also required ;)

**Comment by qaras123** [comment]
Can't ''Run Outside Sandbox'' , even tried back classic version, what am i doing wrong? It just opens the program normally when i use that option, i have been using this method, but now on my new PC it doesn't work, please help.

**Comment by BilleBarrett** [comment]
Did you check these boxes under Global settings



 



(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***

**Reply by qaras123** [reply]
Thank you for your reply. Which settings do i need to change?

**Comment by BilleBarrett** [comment]
<make sure it is checked

 



(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***

**Comment by BilleBarrett** [comment]
Client is having a problem with Firefox

WerFault.exe (8880): SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000001; MsgId: 14 - WerFault.exe [FF000000]

WerFault.exe (4512): SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000001; MsgId: 14 - WerFault.exe [FF000000]

WerFault.exe (5380): SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000001; MsgId: 14 - WerFault.exe [FF000000]

WerFault.exe (12956): SBIE2203 Failed to communicate with Sandboxie Service:  *GUIPROXY_00000001; MsgId: 14 - WerFault.exe [FF000000]

Mozilla Firefox 107.0.1 (x64) (54.77 MB)

 



(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***

**Comment by BilleBarrett** [comment]
I can’t see, to see how to put tis on the pre-release 

But for some reason I cannot set Show Templates for File Recovery

Is there something I did to mess this up

Bill e. Barrett



 



(336) 988-0891    ***@***.***> ***@***.*** 
To subscribe to my FYI emails send an email to
 ***@***.***> ***@***.***

**Comment by 0rigamiManga** [comment]
@DavidXanatos Thanks for continuing the Sandboxie development.  I have been a long-time free user of the product.  Could you please explain the main differences and/or advantages over the Sandboxie-Classic (the one I'm accustomed to using for years) vs. the newer Sandboxie-Plus versions?  I normally do not like to switch to change unless completely necessary.  Any help or insight would be much appreciated.  Thanks in advance!

**Comment by BilleBarrett** [comment]
Can someone tell me why my recover template only opens when I close out Sandboxie ?
[Sandboxie-Plus_inis.zip](https://github.com/sandboxie-plus/Sandboxie/files/10240806/Sandboxie-Plus_inis.zip)
